Task_10/index.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style1.css" type="text/css"/>
+</head>
+<body>
+    <header>
+
+    <div class="conteiner_heder">
+        <img src="./img/rabbit-mouth.png" class="header_rabbit" >
+     <div class="header_text">
+         <span class="orange">orange</span><br>
+         <span class="rabbit_text4" >RABBIT</span>
+     </div>
+    </div>
+    <!-- /////////////////////////// -->
+    <!-- /Гаьбургер меню/ -->
+
+    <div class="hamburger-menu">
+        <input id="menu__toggle" type="checkbox" />
+        <label class="menu__btn" for="menu__toggle">
+          <span></span>
+        </label>
+    
+        <ul class="menu__box">
+                <li><a class="menu__item" href="#">home</a></li>
+                <li><a class="menu__item" href="#">about</a></li>
+                <li><a class="menu__item" href="#">work</a></li>
+                <li><a class="menu__item" href="#">blog</a></li>
+        </ul>
+      </div>
+
+    <!-- \\\\\\\\\\\\\\\\\\\\\\\\\\\ -->
+
+           <div class="icons">
+                <figure>
+            <div class="img1"></div>
+            <figcaption class="figure_text">home</figcaption>
```

Comment:
Please create class for all nav items with common css prop.

Reply:
seems to have fixed